### ISSUE TITLE
Running at only 0.5 fps, CPU only

### ISSUE THREAD

**Issue by mg-12345** [theAIGuysCode/yolov4-deepsort#154]
I'm running this DeepSORT  implementation on Google Colab and the runtime is about 0.4-0.5fps.

I'm connected to a GPU but it seems like only the CPU is working.  I've tried the following:

- Uninstall tensorflow, install tensorflow-gpu 2.3.0 (note: 2.3.0rc0 was a pre-release and it's no longer available)
- Verifying GPU is available before running deepsort
- Converted my darknet weight to TF (several times)
- Restarted the runtime

Any ideas?  Seems like this is an ongoing problem and unfortunately renders this implementation unusable until its fixed.

Update:  Running the command "tf.test.gpu_device_name()" should output "/device:GPU:0" meaning there is one GPU connected and available. With TF 2.3.0 installed, there is no output of this command.  With the latest version of TF installed, the command is recognized and the expected output appears.

The issue here is TF version 2.3.0 seems to not compatible with the current version of CUDA (11.2), so the GPU is not used.  Can this repo be updated to run on a new version of tensorflow?

<img width="149" alt="Capture" src="https://user-images.githubusercontent.com/119070040/204050246-73ce0885-70c3-4350-a82c-9f3ed49e3fe6.PNG">

**Comment by MaximilianoAdaro** [comment]
I'm using **Google Collab** with GPU and this requirements-gpu.txt file changes, and it's working.  But really slow at only 0.6 fps.

```
tensorflow-gpu==2.3.0
opencv-python==4.1.2.30
lxml
tqdm
absl-py
matplotlib
easydict
pillow
numpy==1.18.5
scipy==1.4.1
```

**Comment by mg-12345** [comment]
Yes that requirements file installation TF version 2.3.0 which does not have GPU support for the latest version of CUDA.  It is running with CPU only because TF cannot interface with the GPU.
